manual/Website1/src/App.jsx

```diff
@@ -12,7 +14,10 @@ class App extends Component {
   componentDidMount() {
     corsImport('http://localhost:3002/importManifest.js').then(() => {
       this.setState({ manifestLoaded: true });
-      import(/* webpackIgnore:true */`http://localhost:3002/${window.entryManifest['website-two']['SomeExternalModule.js']}`).then(() => {
+
+      console.log('chunkdeps', getChunkDependencies('http://localhost:3002', 'website-two', 'TitleComponent.js'));
```

Comment:
we need a context provider so react can state things like localhost base path higher up